pcdscalc/be_lens_calcs.py

```diff
@@ -78,20 +83,14 @@ def configure_lens_set_file(lens_file_path):
     Parameters
     ----------
     lens_file_path : str
-        Path to the lens_set file in NumPy .npy format.
-        This is a binary file generated with `numpy.save` which saves an array
-        to a binary file in NumPy .npy format.
+        Path to the lens_set file.
     """
     global LENS_SET_FILE
     if not os.path.exists(lens_file_path):
-        logger.error('Provided invalid path for lens set file: %s',
-                     lens_file_path)
-        return
-    abs_path = os.path.abspath(lens_file_path)
-    file_name = os.path.basename(abs_path)
-    if not file_name.lower().endswith('.npy'):
-        logger.error('Must provide a NumPy .npy format file')
-        return
+        err_msg = ('Provided invalid path for lens set file: %s',
+                   lens_file_path)
+        logger.error(err_msg)
+        raise FileNotFoundError(err_msg)
```

Comment:
The error message here is not substituted correctly, it comes out as a tuple

Reply:
Ohh i see... interesting, thanks for catching this.

Before I commit things though, would something like this be acceptable: 
```
 err_msg = f'Provided invalid path for lens set file: {lens_file_path}'
        logger.error(err_msg)
        raise FileNotFoundError(err_msg)
```
I know we don't like using the `f-Strings` for logging.

Reply:
Yes, personally I've given up on doing the python logging "properly" with the `%s` substitutions because I find myself wanting to re-use the strings... I'm fine with `f-strings` and with `.format`

My understanding is that when you do `logger.error('Bad thing: %s', bad_thing)` and it happens many times you can filter on `Bad thing: %s` to catch all logging messages like this... but unless the log message happens a lot I think this is worth overlooking for formatting clarity/consistency.

@klauer might have an opinion too

Reply:
An opinion? Of course!

Use:
* `logging.error('%s', value)` if the string isn't going to be reused (generally speaking, I think this is the majority of the cases)
* `f'{value}'` for _pretty much_ everything else. 

`.format()` most of the time ends up not being as nice as an f-string, as it requires you to visually match up the arguments instead of seeing how they're going to be interpolated directly in the string. If you find yourself justifying the use of `.format()` over an f-string because you need to do some calculation/cleanup before formatting, I'd recommend just making a temporary variable and using an f-string with that.

Reply:
Thank you, this should be fixed in ef27fed